project_update_visible/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    # Copyright 2023 Binhex (https://www.binhex.cloud)
```

Comment:
The copyright should be on top of the file

```python
# Copyright 2023 Binhex (https://www.binhex.cloud)
# Copyright 2023 Binhex (https://www.binhex.cloud)
{
....
```

Check for example any other module from the repository :wink: